views/sections/library.html

```
@@ -2,7 +2,7 @@
   <div class="flex flex-col md:flex-row md:items-center md:justify-between gap-4 mb-8">
     <h1 class="text-3xl font-bold text-slate-800 dark:text-white">Music Library</h1>
     <div class="flex items-center gap-3">
-      <button hx-get="/ui/library" hx-swap="outerHTML" hx-target="#contenido" title="Refresh Library View"
+      <button hx-get="/library" hx-swap="outerHTML" hx-target="#contenido" title="Refresh Library View"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)